pkg/crc/cluster/cluster.go

```diff
@@ -87,7 +88,24 @@ func GetRootPartitionUsage(sshRunner *ssh.Runner) (int64, int64, error) {
 	return diskSize, diskUsage, nil
 }
 
-func AddPullSecretInTheCluster(ocConfig oc.Config, pullSec *PullSecret) error {
+func EnsurePullSecretPresentInTheCluster(ocConfig oc.Config, pullSec *PullSecret) error {
+	if err := WaitForOpenshiftResource(ocConfig, "secret"); err != nil {
+		return err
+	}
+
+	stdout, _, err := ocConfig.RunOcCommandPrivate("get", "secret", "pull-secret", "-n", "openshift-config", "-o", `jsonpath="{['data']['\.dockerconfigjson']}"`)
+	if err != nil {
+		return err
+	}
+	decoded, err := base64.StdEncoding.DecodeString(stdout)
+	if err != nil {
+		return err
+	}
+	if err := validation.ImagePullSecret(string(decoded)); err == nil {
+		return nil
+	}
+
```

Comment:
Should this check that `string(decoded)` is the same as `pullSec.Value()`?

Reply:
If we do that, each time we do a `crc start` it will ask for the pull secret.